src/Symfony/Component/VarDumper/Tests/Cloner/VarClonerTest.php

```diff
@@ -379,6 +382,59 @@ public function testJsonCast()
         $this->assertStringMatchesFormat(\PHP_VERSION_ID >= 70200 ? str_replace('"1"', '1', $expected) : $expected, ob_get_clean());
     }
 
+    /**
+     * @requires extension gmp
+     */
+    public function testGmpCast()
```

Comment:
but in fact, this should not be in VarClonerTest
the existing test in the caster in enough IMO